docs/UserGuide.md

```diff
@@ -288,6 +295,50 @@ Suppose the following patients were added.
 * `viewTask 2` will display:
   * `Change dressing on left arm`
 
+### Adding a medical condition: `addCondition`
+
+_Command syntax to be updated ..._
+
+Format: `addCondition PATIENT_INDEX c/CONDITION`
```

Comment:
Is there a reason PATIENT_INDEX does not use the cli syntax of -p?

Reply:
I think Rama1 was supposed to update this part, since the parsing of `-p` and similar flags only falls under `AddGenericCommandParser`, which currently only supports the adding of tasks and patients. This is the part in `AddGenericCommandParser` that handles `-p`:

```java
if (patientIndex.isPresent()) {
    return new AddTaskCommandParser().parse(patientIndex.get() + " " + args);
}
```
I am inclined to think that some sort of switch statement to support attributes like conditions, medications, remarks, etc. would be required. But that warrants a way to check what exactly we are trying to add, since `AddGenericCommandParser` only parses the index-based parameters before passing parsing of text-based params to the respective commands.

I think it was mentioned before that we implement addition/deletion/editing of the attributes involving the use of `GenericList` with the old command words first, then Rama1 would integrate them with the generic versions.